Capstone-Projects-2022-Spring/project-vicsr
# Vic 154 UI render selected document with each page as an image with ability to add drawing highlighting capability later

Pages of documents are rendered as an image which is able to be drawn upon. Each page can be accessed with boundary checking (i.e., can not go into negative pages or into more pages than the document has). 

Testing instructions:
- Log into the gramgram@gmail.bob account
- click through documents (the first three documents were uploaded before the changes made to the backend, so they will break the page, don't click them) to see that you can switch the document you are accessing. 
- Click document with ID 23
- Click previous page, observe nothing happens
- click next, observe that you are brought to the next page
- Feel awesome and cool 

In the future we will need to save/load and unload highlights on page/document change
